bugwarrior/services/bitbucket.py

```diff
@@ -204,8 +204,10 @@ def get_annotations2(self, tag, issue, issue_obj, url):
         )
 
     def get_owner(self, issue):
-        tag, issue = issue
-        return issue.get('responsible', {}).get('username', None)
+        _, issue = issue
+        assignee = issue.get('assignee', None)
+        if assignee is not None:
+            return assignee.get('username', None)
```

Comment:
@ryneeverett the behavior is slightly different: the one-liner crashes if `'assignee'` is a valid key in `issue` but the value is `None` (which does happen in the data returned by bitbucket).